packages/uxpin-merge-cli/src/steps/serialization/vcs/repositories/git/util/isGitRepository.ts

```diff
@@ -1,10 +1,15 @@
+import debug from 'debug';
+
 import { execAsync } from '../../../../../../utils/child_process/execAsync';
 
+const log = debug('uxpin');
+
 export async function isGitRepository(cwd: string): Promise<boolean> {
   try {
     await execAsync('git status', { cwd });
     return true;
-  } catch (e) {
+  } catch (error) {
+    log(`git status failed`, (error as Error).message);
```

Comment:
Can we maybe remove the `git status failed`? Right now when we get the error message, it says

`git status failed Command failed: git status` which almost sounds a bit repetitive. This sounds a bit more precise and easy to follow `Command failed: git status`

What do you think?

Reply:
I wanted an error message that gives the context it's not the Git command itself that breaks, but the `execAsync` used to run a shell from Node.js.

Something like "Unable to call Git command:", followed by the error message would be better?

Reply:
Ok I understand now, thank you for explaining! In that case, for context I think `Unable to call Git command:` sounds better. Gives better context. 👍

Reply:
Thinking more , we have already a final `ERROR Could not determine...` at the end, always printed (whether it's debug or not), so no need to be too verbose, you are right.

So we can do this:

- Show a message before the check
- Show only the raw error message if it fails, like this:

![image](https://github.com/UXPin/uxpin-merge-tools/assets/5546996/14bbcf49-90fa-48a3-aee0-f6aa8f6b668d)